gradle.properties

```diff
@@ -33,23 +33,3 @@ org.gradle.java.installations.auto-download=false
 
 # Enable the maven local repository (for local development when needed) when present (value ignored)
 #enableMavenLocalRepo = true
-
-# The default Hibernate ORM version (override using `-PhibernateOrmVersion=the.version.you.want`)
-hibernateOrmVersion = 7.0.2.Final
-
-# Override default Hibernate ORM Gradle plugin version
-# Using the stable version because I don't know how to configure the build to download the snapshot version from
-# a remote repository
-#hibernateOrmGradlePluginVersion = 7.0.2.Final
-
-# If set to true, skip Hibernate ORM version parsing (default is true, if set to null)
-# this is required when using intervals or weird versions or the build will fail
-#skipOrmVersionParsing = true
```

Comment:
I don't see this property mentioned anywhere anymore

Reply:
Is it still needed?
IIUC, if `skipOrmVersionParsing ` is false, `hibernateOrmVersion` is parsed and may raise if version has no `.` in it.
https://github.com/hibernate/hibernate-reactive/blob/c254ebb5a6cda29843b8a3c1a8d6f8713450eec5/local-build-plugins/src/main/java/org/hibernate/reactive/env/ProjectVersion.java#L17-L24

If it is needed to test "weird versions", why not simply use that version ??? 

`❯ ./gradlew compileJava -PhibernateOrmVersion=1234weiredversion`

Reply:
I thought I answered this, but it seems I dreamt it.

The reason the property and these checks exist is to prevent releasing a library with the wrong version or pushing the wrong version upstream.